Library/Homebrew/dev-cmd/pr-publish.rb

```diff
@@ -16,6 +16,10 @@ def pr_publish_args
       EOS
       flag   "--tap=",
              description: "Target tap repository (default: `homebrew/core`)."
+      flag   "--workflow=",
+             description: "Target workflow filename (default: `publish-commit-bottles.yml`)."
```

Comment:
When would you use a different name here?

Reply:
Probably for other third-party taps, this option might be useful. I'm going to actually use it in mine.

Reply:
Cool. I wonder if it can be inferred somehow 🤔. Don't let that block this PR.

Reply:
Hmm, nothing that I am aware of as of now, but I get the idea.